bedrock/settings/static_media.py

```diff
@@ -798,24 +746,22 @@
     'styleguide-docs-mozilla-accordion': {
         'source_filenames': (
             'css/base/mozilla-accordion.less',
-            'css/sandstone/sandstone-resp.less',
         ),
         'output_filename': 'css/styleguide-docs-mozilla-accordion-bundle.css',
     },
     'styleguide-docs-mozilla-pager': {
         'source_filenames': (
-            'css/sandstone/sandstone-resp.less',
             'css/styleguide/docs/mozilla-pager.less',
         ),
         'output_filename': 'css/styleguide-docs-mozilla-pager-bundle.css',
     },
     'styleguide-docs-send-to-device': {
         'source_filenames': (
-            'css/sandstone/sandstone-resp.less',
             'css/base/send-to-device.less',
         ),
         'output_filename': 'css/styleguide-docs-send-to-device-bundle.css',
     },
+    # no longer used on bedrock. possibly referenced on external sites? should investigate.
```

Comment:
Yep we need to keep this around as long as external sites may still be using tabzilla. I think we really need to set a public date for deprecation and let people know, before we remove this from bedrock.